src/Controller/Sign.php

```diff
@@ -27,27 +27,36 @@
 
 final class Sign
 {
-    public function __invoke(EnvelopeBuilderInterface $envelopeBuilder, SignatureExtractor $signatureExtractor, Request $request, EventDispatcherInterface $eventDispatcher, LoggerInterface $logger): Response
+    private $envelopeBuilder;
+    private $signatureExtractor;
+
+    public function __construct(EnvelopeBuilderInterface $envelopeBuilder, SignatureExtractor $signatureExtractor)
```

Comment:
in the constructor?

Reply:
with the interface and the aliases, it suppose you can let the autowiring do the job right? Unless you cannot because of the service name.
WDYT?

Reply:
oh right no, never mind.
You've got 2 signatures definitions...

Reply:
Need to inject the correct services, otherwise the autowiring will inject the "default" one, which may use the wrong configuration